windows-nsis/openvpn.nsi

```diff
@@ -244,11 +244,12 @@ Section -pre
 
 		; Stop & Remove previous OpenVPN service
 		DetailPrint "Removing any previous OpenVPN interactive service..."
-		nsExec::ExecToLog /OEM '"$INSTDIR\bin\openvpnserv.exe" -remove'
+		SimpleSC::RemoveService OpenVPNService
```

Comment:
Should be "SimpleSC::RemoveService OpenVPNServiceInteractive" isn't it ? 
OpenVPNService has to be removed in the secService if selected.